hbshelpers/chainedLookup.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * A Handlebars helper that performs multiple lookups in a row.
+ *
+ * @param {Object} context The context to perform lookup on
+ * @param {...string} lookupChain The lookups to perform.
+ * @param {import('handlebars').HelperOptions} options
+ * @returns {any} The result of the lookup
+ */
+module.exports = function chainedLookup(context, ...lookupChain) {
+  let lookupResult = context;
+  for (const lookup of lookupChain.slice(0, -1)) {
+    if (!lookupResult || typeof lookupResult !== 'object') {
+      return '';
```

Comment:
re: Returning empty string as the fail case -- I could see there being a time we want to know that it exists and it's just empty vs. the object is malformed. Do we want to return undefined here instead?

Reply:
yeah makes sense!